src/docs/get-started/install/_get-sdk-win.md

```diff
@@ -12,25 +19,52 @@
     (for example, `C:\src\flutter`; do not
     install Flutter in a directory like `C:\Program Files\` that requires
     elevated privileges).
- 1. Locate the file `flutter_console.bat` inside the `flutter` directory.
-    Start it by double-clicking.
+{% endcomment %}
+
+<ol markdown="1">
+<li markdown="1">At the command line,
+    go to the desired installation location
+    for the Flutter SDK (for example, `C:\src\flutter`; 
+    do not install Flutter in a directory like `C:\Program Files\`
+    that requires elevated privileges).
+</li>
+<li markdown="1">Clone the repo using `git clone`.
+    The following example clones the stable channel:
+
+```sh
+git clone -b stable https://github.com/flutter/flutter.git
+```
+</li>
+<li markdown="1">In the Finder, navigate to the `flutter`
+    directory and locate the `flutter_console.bat` file.
+    Double-click the file to start it.
+</li>
+</ol>
 
 You are now ready to run Flutter commands in the Flutter Console!
 
-To update an existing version of Flutter,
-see [Upgrading Flutter](/docs/development/tools/sdk/upgrading).
+{{site.alert.warning}}
```

Comment:
I find this confusing. We haven't asked them to download a zip or unzip anything, so there's no real context for them to understand this comment. I vote to drop it.

Reply:
Yeah, agree -- that explains why _we_ don't go down that route, but the user doesn't need to worry about what we've discarded.